live-examples/js-examples/object/object-freeze.html

```diff
@@ -1,14 +1,14 @@
 <pre>
-<code id="static-js">const object1 = {
+<code id="static-js">const object = {
   property1: 42
 };
 
-const object2 = Object.freeze(object1);
+Object.freeze(object1);
```

Comment:
The variable declared at line 2 is renamed to `object` but this isn't, so we'll get an error here.

I think we should keep the name `object1` - I'm uncomfortable with a variable named `object`, and using the suffix is recommended in the style guide: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#numerical-suffixes.

Reply:
How about `obj` ?

Reply:
Yes, I'd be happy with that too :).